packages/core/src/data-grid-dnd/data-grid-dnd.tsx

```diff
@@ -32,6 +33,8 @@ const DataGridDnd: React.FunctionComponent<DataGridDndProps> = p => {
     const {
         onColumnMoved,
         onColumnResized,
+        onColumnResizedStart,
```

Comment:
These need to be mangled in data-editor.tsx so they account for the rowMarkerOffset.

Reply:
Could you elaborate what you mean by that? 

The normal `onColumnResized` doesn't seem to be used in `data-editor.tsx`.
https://github.com/glideapps/glide-data-grid/blob/main/packages/core/src/data-editor/data-editor.tsx

For `onColumnResizedStart`, I just use the `args.bounds.width`. For `onColumnResizedEnd`, ` lastResizeWidthRef.current`.

Reply:
Sorry my brain was on autopilot here. This takes the actual column not the index. This presents a different issue of sorts but one that is pre-existing. The auto column sizer will now make this feed back a different object that what was fed in by the user. Thought is needed here, but not a blocker for this PR. Can you please retarget to 4.0.3?